docs/zone_outage.md

```diff
@@ -0,0 +1,23 @@
+### Zone outage scenario
+Scenario to create outage in a targeted zone in the public cloud to understand the impact on both Kubernetes/OpenShift platform as well as applications running on the worker nodes in that zone. It tweaks the network acl of the zone to simulate the failure and that in turn will stop both ingress and egress traffic from all the nodes in a particualar zone for the specified duration and reverts it back to the previous state. Zone outage can be injected by placing the zone_outage config file under zone_outages option in the [kraken config](https://github.com/cloud-bulldozer/kraken/blob/master/config/config.yaml). Refer to [zone_outage_scenario](https://github.com/openshift-scale/kraken/blob/master/scenarios/zone_outage.yaml) config file for the parameters that need to be defined.
+
+Refer to [cloud setup](cloud_setup.md) to configure your cli properly for the cloud provider of the cluster you want to shut down
+
+##### Current accepted cloud types:
+* [AWS](cloud_setup.md#aws)
+
+##### Sample scenario config
+```
+zone_outage:                                         # Scenario to create an outage of a zone by tweaking network ACL
+  cloud_type: aws                                    # cloud type on which Kubernetes/OpenShift runs. aws is only platform supported currently for this scenario.
+  duration: 600                                      # duration in seconds after which the zone will be back online
+  vpc_id:                                            # cluster virtual private network to target
+  subnet_id:                                         # subnet-id to deny both ingress and egress traffic
+```
```

Comment:
Might be nice to add the 2nd bullet point below here to help set the vpc id and subnet id for the config

Reply:
Fixed it, thanks.